uber/templates/preregistration/shirt_reorder.html

```diff
@@ -1,4 +1,4 @@
-{% extends "base.html" %}
+{% extends "base-admin.html" %}
```

Comment:
this seems like it might be a mistake, the prereg pages are public-facing and probably shouldn't inherit from any admin stuff